nginx/default.conf

```diff
@@ -6,6 +6,7 @@ server {
     ssl_certificate /etc/nginx/cert.pem;
     ssl_certificate_key /etc/nginx/private.key; 
 
+    client_max_body_size 8M;
```

Comment:
(putting an inline comment now so you see it) The upload restrictions should be noted on the upload form so users know (file size, logged in, caption, etc)